ttcc/core.py

```diff
@@ -4,36 +4,56 @@
 
 def register(device_name, device_details):
     global DEVICES
+
     # Overwrite or ignore a duplicate registration?
     # Check if the device details is in the correct format
     # Check if type is an object of a subclass of TypeBaseClass
-    DEVICES[device_name] = device_details
-
+    
 def parse_device(sentence):
-    pass
+    global DEVICES
+    words = sentence.split()
+    target_device = []
+
+    target_device = list(set(words)&set(DEVICES.keys()))
+    for device in DEVICES:
+            device_alias = list(set(words) & set(DEVICES[device]['alias']))
+            target_device.extend(device_alias)
```

Comment:
Appending the device name should be enough. So it'll be `target_device.append(device)`. We don't need information on what alias what used in the sentence, knowing the device is enough.